### PR TITLE
fixed search listing shortcode atts not working after ajax load

### PR DESCRIPTION
**Pull request by syedgalib**

### CODE REVIEW

**Review by vairafiq — APPROVED** [review]
Tested. Works fine

File: includes/classes/class-shortcode.php [hunk 1]
```diff
@@ -179,6 +179,9 @@ public function all_locations( $atts ) {
 	}
 
 	public function search_listing( $atts ) {
+
+		// var_dump(  $atts );
```
Comment (kowsar89) [inline includes/classes/class-shortcode.php:183]: remove var_dump

**Review by kowsar89 — CHANGES_REQUESTED** [review]

**Review by kowsar89 — CHANGES_REQUESTED** [review]
Doesn't work if shortcode is used multiple times in a page.
image

**Comment by syedgalib** [comment]
fixed

**Pull request merged**